### DISCUSSION TITLE
Directly delete short scenes by Python API

### DISCUSSION THREAD

**Discussion by yumianhuli1** [Breakthrough/PySceneDetect#422] [General]
Hello！
 The min_scene_len parameter specified in scene_manager.add_detector(ContentDetector(threshold=27.0, min_scene_len=15)) will only merge short scenes into large scenes. However, my requirement is to directly delete scenes with fragments less than 0.5 seconds without appearing in the scene list. Do you have this api function and parameter? Or Just use min_scene_len=0 and then use ffmpeg?
 Thank you!

**Comment by Breakthrough** [comment]
You can filter them out pretty easily with `min_scene_len=0` and a list comprehension:

```python
import scenedetect
detector = scenedetect.ContentDetector(min_scene_len=0))
scenes = scenedetect.detect("tests/resources/goldeneye.mp4", detector=detector)
# drop scenes shorter than 0.5 seconds, s[0] is start timecode, s[1] is end timecode
scenes = [s for s in scenes if (s[1] - s[0]) >= 0.5]

**Reply by yumianhuli1** [reply]
> You can filter them out pretty easily with `min_scene_len=0` and a list comprehension:
> 
> ```python
> import scenedetect
> detector = scenedetect.ContentDetector(min_scene_len=0))
> scenes = scenedetect.detect("tests/resources/goldeneye.mp4", detector=detector)
> # drop scenes shorter than 0.5 seconds, s[0] is start timecode, s[1] is end timecode
> scenes = [s for s in scenes if (s[1] - s[0]) >= 0.5]
> ```

thanks！
